fleet/lib/openvpn-db/fleet.yaml

```diff
@@ -0,0 +1,19 @@
+name: openvpn-db
+defaultNamespace: &name openvpn-db
+labels:
+  bundle: *name
+dependsOn:
+  - name: mariadb-operator
+    namespace: mariadb-system
+  - name: openvpn-db-pre
+    namespace: openvpn-db
+targetCustomizations:
+  - name: ayekan
```

Comment:
What, if anything, would be different for the ayekan deployment than a production deployment?

Reply:
There is not much difference for now, but it would be good to keep it as is to test any new changes that we might add in the future to dev.